### PR TITLE
Handle @include in template files

### PR DESCRIPTION
**Pull request by cycomachead**
Same tweaks as the other PR. :)

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -66,19 +65,34 @@ function build() {
...
+                 done < "templates/$template.tmp"
+                 # append to the temporary HTML file, evaluating any possible params
+                 envsubst < .template.html >> tmp.html
+                 rm -f .template.html
             done
 
             if grep -q @content $html; then
                 # replace the @content string for the contents of the template file
-                sed -e '/@content/ {' -e 'r tmp.html' -e 'd' -e '}' -i $sed_input_extras $html
+                if [[ "$OSTYPE" == "linux-gnu" ]]; then
```
Comment (cycomachead) [inline build.sh:86]: Oh I also had to do this very sad change. 

doing `-i $var_name` with sed on the Mac leads to it sticking `""` at the end of files as the backup extension. Sigh. :(

**Comment by bromagosa** [comment]
Thanks! Very nice addition. I'll test it later today if I have the time :)

**Comment by cycomachead** [comment]
Cool thanks! :)

It's definitely not yet recursive, though at some point it'd be nice if it were, but I figure this should be enough for a while.

**Comment by bromagosa** [comment]
It's way better than having to split files in two :+1:

**Comment by bromagosa** [comment]
Just one last tiny remark. Could you please modify `README.md` to reflect this change? I can update the example page myself afterwards.

Thanks a lot!

**Pull request merged**